apps/console/src/features/organizations/pages/organization-roles-edit.tsx

```diff
@@ -0,0 +1,85 @@
+import { PageLayout } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { AppConstants, AppState, FeatureConfigInterface, history } from "../../core";
+import { getOrganizationRoleById } from "../api";
+import { EditOrganizationRole } from "../components/edit-organization-role";
+import { OrganizationInterface, OrganizationRoleInterface } from "../models";
+
+const OrganizationRolesEdit: FunctionComponent = (
```

Comment:
Let's add a comment describing the component

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/81488c55dd6c9c7a63575dee78e71f2b80643963